spec/features/cards_spec.rb

```diff
@@ -1,23 +1,54 @@
 require 'support/login_helper'
 require 'rails_helper'
 
-describe 'filling the form' do
+describe 'Manipulating cards and decks' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.